src/main/scala/fundamentals/level04/LogParser.scala

```diff
@@ -50,6 +50,15 @@ object LogParser {
    * 2. Comment out the contents of src/test/scala/fundamentals/level04/Types.scala
    */
 
+  /**
+    * We want to parse log messages. In order to do so, we need a safe way to parse integers in those messages.
+    *
+    * One possibility is to define a function that converts a `String` to an `Option[Int]`
+    *
+    * Hint: You may copy `parseIntSafe` and `tryToOption` from TryExercises
```

Comment:
Maybe the hint can be `You've done this before in TryExercises` to force them to think :)

Reply:
`tryToOption` isn't in `TryExercises` so might be worth rephrasing.

```suggestion
    * Hint: You may copy `parseIntSafe` from TryExercises and use `tryToOption` from above.
```

Reply:
But `tryToOption` is in `TryExercises`. Would you like me to move it to `LogParser`?

Reply:
Oh my I'm sorry, I thought `tryToOption` was here lol.